src/Webapi/Webapi__Navigator.res

```diff
@@ -1,25 +1,26 @@
-type t
-
-/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorid */
-
-@get external userAgent: t => string = "userAgent"
-@get external vendor: t => string = "vendor"
-
-/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorlanguage */
-
-/**
- * Returns a BCP 47 language tag
- */
-@get external language: t => string = "language"
-
-/**
- * Experimental. Returns a readonly array of BCP 47 language tags
- */
-@get external languages: t => array<string> = "languages"
-
-/* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatoronline */
-
-@get external onLine: t => bool = "onLine"
+type t = private {
+  /* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorid */
+  userAgent: string,
+  vendor: string,
+  /**
+   * Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorlanguage
+   *
+   * Returns a BCP 47 language tag
+   */
+  language: string,
+  /**
+   * Experimental. Returns a readonly array of BCP 47 language tags
+   */
+  languages: array<string>,
+  /* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatoronline */
+  onLine: bool,
+  /* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorcookies */
+  cookieEnabled: bool,
+  /* Spec: https://html.spec.whatwg.org/multipage/system-state.html#navigatorplugins */
+  pdfViewerEnabled: bool,
+  /* Spec: https://html.spec.whatwg.org/multipage/workers.html#navigatorconcurrenthardware */
+  hardwareConcurrency: float,
```

Comment:
Should this be `int` for convenience I see in the spec that it's defined as a long but 2147483647 seems like a lot of cores and floats is annoying in this regard when the value is likely to be something you want to loop on to generate say worker threads.

Reply:
It was already float, so I'm curious what @illusionalsagacity thinks, but I agree int is a wide enough number for the foreseeable future.

Reply:
I think when I was testing in various browsers, some properties that were defined as a `long` in the spec, even for things that should "obviously" be ints, ended up being floats. I may even have had a few of them as int before, but when I ran into issues I reverted back to float.

So I'd just encourage some cross browser testing before changing it.

Reply:
With JS specs it's always just "number" which could be anything. Unless some browsers return fractional numbers `int` will be safe until we have computers with 2 billion cores 😂 

Firefox and Chrome both return whole numbers across both windows and mac, Safari doesn't support it yet. I think this is safe at least for now.